src/BlockPatterns.php

```diff
@@ -226,10 +229,17 @@ public function register_block_patterns() {
 				$pattern_data['description'] = translate_with_gettext_context( $pattern_data['description'], 'Pattern description', 'woo-gutenberg-products-block' );
 			}
 
+			$pattern_data_from_dictionary = $this->get_pattern_from_dictionary( $dictionary, $pattern_data['slug'] );
+
 			// The actual pattern content is the output of the file.
 			ob_start();
+			if ( ! is_null( $pattern_data_from_dictionary ) ) {
+				$content = $pattern_data_from_dictionary['content'];
+				$images  = $pattern_data_from_dictionary['images'] ?? array();
+			}
```

Comment:
From what I understand, this is a way to valorize the variable $content and $images from the outside, right? If I'm right, it is a tricky but cool thing for this scope! Do you have some resources about this technique (if you have it otherwise, I can search it)?

I would suggest adding a comment about this approach because those who don't work a lot with PHP could have some issues understanding this approach.

Reply:
I don't really have any resources about it, sorry! 😅 
I'll add a comment, makes sense!